tests/infra/assert_mixin.py

```diff
@@ -62,7 +67,16 @@ def assertResponse(
                 kwargs['headers'] = {}
             kwargs['headers']['Content-Type'] = "application/json"
 
-        response = getattr(self.app, method)(path, **kwargs)
+        for ix in range(redirect_follow_retry + 1):
+            response = getattr(self.app, method)(path, **kwargs)
+            if ix == redirect_follow_retry or response.status_code != requests.codes.moved:
+                break
+            retry_after = int(response.headers['Retry-After'])
+            self.assertGreaterEqual(retry_after, min_retry_interval_header)
+            url = response.headers['Location']
+            builder = UrlBuilder(url).set(scheme="", netloc="")
```

Comment:
Why no scheme?

Reply:
URLs passed to the the unit test chalice server only have the path.  /v1/files, for instance.